api/files/serializers.py

```diff
@@ -47,6 +47,14 @@ def to_internal_value(self, data):
         except StopIteration:
             self.fail('invalid_data')
 
+    def to_representation(self, value):
```

Comment:
I think this is going to fail JSON-API schema validation. If there's no checkout url the href comes back null which will fail validation. I think if you do a None check on url and then just `return {}` it will validate and still have the expected behavior.